src/dp.rs

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Differential privacy (DP) primitives.
+use crate::field::FieldElement;
+use crate::vdaf::prg::SeedStream;
+use std::fmt::Debug;
+
+/// Positive rational number to represent DP parameters and noise distributions.
```

Comment:
I think it's worth noting here why we think a floating point number shouldn't be used (eventually we're going to need to encode this value in a protocol message).